auctioneerrpc/auctioneer.proto

```diff
@@ -373,6 +373,11 @@ enum ChannelType {
     outputs that pay directly to the channel initiator (the seller).
     */
     SCRIPT_ENFORCED_LEASE = 2;
+
+    /*
+    A channel type that uses a Pay-to-Taproot output for the funding output.
+    */
+    SIMPLE_TAPROOT = 3;
```

Comment:
Yes, you are correct... Unfortunately we started this file/enum before we were aware of that. So in this specific case I'd vote for consistency with the existing enum names rather than having one name being different than the others (we can't rename the existing ones without breaking older gRPC or REST clients).